sentinel-core/src/main/java/com/alibaba/csp/sentinel/traffic/rule/traffic/router/Route.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.traffic.rule.traffic.router;
+
+import java.util.List;
+
+import com.alibaba.csp.sentinel.traffic.rule.traffic.router.match.StringMatch;
+
+/**
+ * @author panxiaojun233
+ */
+public class Route {
+	private String name;
+	private List<StringMatch> services;
+	private List<RouteDetail> routedetail;
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public List<StringMatch> getServices() {
+		return services;
+	}
+
+	public void setServices(List<StringMatch> services) {
+		this.services = services;
+	}
+
+	public List<RouteDetail> getRoutedetail() {
+		return routedetail;
+	}
+
+	public void setRoutedetail(List<RouteDetail> routedetail) {
+		this.routedetail = routedetail;
+	}
+
+	@Override
+	public String toString() {
+		return "DubboRoute{" +
```

Comment:
Please re-generate `toString()` for this.

Reply:
done